src/main/flight/mixer.c

```diff
@@ -105,18 +105,24 @@ static FAST_DATA_ZERO_INIT float motorRangeMax;
 static FAST_DATA_ZERO_INIT float motorOutputRange;
 static FAST_DATA_ZERO_INIT int8_t motorOutputMixSign;
 
-
 static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 {
     static uint16_t rcThrottlePrevious = 0;   // Store the last throttle direction for deadband transitions
     static timeUs_t reversalTimeUs = 0; // time when motors last reversed in 3D mode
     static float motorRangeMinIncrease = 0;
 
+#ifdef USE_DYN_IDLE
+    if (!mixerRuntime.feature3dEnabled && mixerRuntime.dynIdleMinRps && isDshotTelemetryActive()) {
```

Comment:
I will mark this as resolved because I have implemented the changes as requested.

I am not entirely happy with making this change, but we will find out if my concerns are justified as people fly it.

The requested change assumes that DShot Telemetry is always working properly if configured to be on.  

Of course, it may not be.  A user could turn Dynamic Idle on and enable DShot telemetry, but not actually check that it is working properly.

In the situation where RPM filtering is configured to be on, a failure of DShot Telemetry will prevent motors from spinning.  This would also prevent issues related to dynamic idle, if it was configured to be on, since the quad would not take off.

But in the event that the user is *not* using RPM Filtering, but has Dynamic Idle configured to be on, and DShot Telemetry configured to be on, a failure of DShot Telemetry could cause unpredictable results.  Most likely the quad would tend to desync or the motors would barely spin on arming.  This would alert the user to a failure of their telemetry, and is of itself not such a bad thing.

We will need to see if this is a real problem or not.  It is a very hard thing to test, since I have no way to reliably cause DSHot Telemetry to fail.